hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -845,9 +845,6 @@ private ResponseCodeEnum sanityChecked(
 	}
 
 	private ResponseCodeEnum validateAndAutoAssociate(AccountID aId, TokenID tId) {
-		if (hederaLedger.maxAutomaticAssociations(aId) > 0) {
-			return associate(aId, List.of(tId), true);
-		}
```

Comment:
Can we add a comment here that says something like:
`// Issue-2286: Prevent automatic associations.`
`// When automatic associations is ready, check here whether to auto-associate`

Reply:
Added.